dlf/plugins/navigation/locallang.xml

```diff
@@ -56,6 +57,7 @@
 			<label index="zoom-in">Ansicht vergrößern</label>
 			<label index="zoom-out">Ansicht verkleinern</label>
 			<label index="zoom-fullscreen">Vollansicht</label>
+			<label index="addBasket">Zum Warenkorb hinzufügen</label>
```

Comment:
Please remove those unused language entries for the navigation plugin. Since there are no code changes to the plugin itself, why would we need those language entries?